trax/models/research/bert.py

```diff
@@ -71,8 +71,39 @@ def BERTRegressionHead():
       tl.Fn('RemoveAxis', lambda x: np.squeeze(x, axis=1))
   ])
 
+@gin.configurable()
+def BERTMLMHead(vocab_size=30522):
+  return tl.Serial([
+      tl.Select([1], n_in=2),
+      tl.Dense(vocab_size,
+               kernel_initializer=tl.RandomNormalInitializer(0.02),
+               bias_initializer=tl.RandomNormalInitializer(1e-6),
+               ),
+  ])
+
+@gin.configurable()
+def BERTPretrainingLoss():
```

Comment:
Can you split the losses, that is make each of them optional?

Reply:
Losses are already optional (You can use just MLMhead or just ClassifierHead with WeightedCrossEntropy loss and it will work). However I did not implement choosing random sentence from corpus document for MLM training without adding NSP loss. I'll add it and add gin file for training only MLM as an example